arkouda/pdarrayclass.py

```diff
@@ -1384,6 +1383,53 @@ def save(
         else:
             raise ValueError("Supported file formats are 'HDF5' and 'Parquet'")
 
+    @typechecked
+    def to_parquet(
+        self, prefix_path: str, dataset: str = "array", mode: str = "truncate", compressed: bool = False
+    ) -> str:
+        from arkouda.io import mode_str_to_int
+
+        return cast(
+            str,
+            generic_msg(
+                cmd="writeParquet",
```

Comment:
Just something to think about - Might be good to change this cmd string to "toParquet" so it remains consistent with "tohdf" or change "tohdf" to "writeHDF".

Reply:
@joshmarshall1 - Noted. I will add this to my list of things moving forward. I did not want to make any server side modifications for this PR.